driver.go

```diff
@@ -257,10 +258,114 @@ func findNeighbors(tx neo4j.Transaction, ids []uint64) ([]neo4jNode, []neo4jNode
 	return sources, targets, edges, nil
 }
 
-func MakeProperties(input map[string]interface{}) map[string]*ls.PropertyValue {
+func MakePropertiesForSetNodeValue(cfg Config, node graph.Node, input map[string]interface{}) interface{} {
```

Comment:
SetNodeValueAfterLoad

Write some comments saying this should be calledafter load, after setNodeProperties, and that input has neo4j native values